firebase/test/core/data/schemas/memo.spec.ts

```diff
@@ -0,0 +1,127 @@
+import Ajv2020 from "ajv/dist/2020";
+import { SchemaValidator } from "#data/schemas/schema-validator";
+import { doesNotThrow, throws } from "assert";
+import SerializationError from "#faults/errors/serialization-error";
+
+describe("Memo Schema Validation", () => {
+  const validator = new SchemaValidator(new Ajv2020());
+  const allowedProperties = ["bold", "italic", "underline", "code-block"];
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  let rawMemo: any;
+
+  beforeEach(() => {
+    rawMemo = {
+      uniqueId: "any",
+      question: [
+        {
+          insert: "Question string",
+        },
+      ],
+      answer: [
+        {
+          insert: "Answer string",
+        },
+      ],
+    };
+  });
```

Comment:
why not transform this into a `_newRawMemo()` function that returns the referred json? It gives a much more direct approach and don't have to manipulate a "wider-scope" variable

Reply:
Agreed